api/checks/webhook.go

```diff
@@ -204,20 +231,50 @@ func handleCheckRunEvent(aeAPI shared.AppEngineAPI, checksAPI API, payload []byt
 			return false, nil
 		}
 	}
-	if !shouldSchedule {
-		log.Debugf("Ignoring %s action for %s check_run", action, status)
+	if shouldSchedule {
+		name, sha := checkRun.GetCheckRun().GetName(), checkRun.GetCheckRun().GetHeadSHA()
+		log.Debugf("GitHub check run %v (%s @ %s) was %s", checkRun.GetCheckRun().GetID(), name, sha, action)
+		spec, err := shared.ParseProductSpec(checkRun.GetCheckRun().GetName())
+		if err != nil {
+			log.Errorf("Failed to parse \"%s\" as product spec", checkRun.GetCheckRun().GetName())
+			return false, err
+		}
+		checksAPI.ScheduleResultsProcessing(sha, spec)
+		return true, nil
+	}
+	log.Debugf("Ignoring %s action for %s check_run", action, status)
+	return false, nil
+}
+
+func handleAzurePipelinesEvent(log shared.Logger, event checkRunEvent) (bool, error) {
+	status := event.GetCheckRun().GetStatus()
+	if status != "completed" {
+		log.Infof("Ignoring non-completed status %s", status)
 		return false, nil
 	}
+	detailsURL := event.GetCheckRun().GetDetailsURL()
+	buildID := extractAzureBuildID(detailsURL)
+	if buildID == "" {
+		log.Errorf("Failed to extract build ID from details_url \"%s\"", detailsURL)
+		return false, nil
+	}
+	artifact := fmt.Sprintf(
+		"https://dev.azure.com/%s/%s/_apis/build/builds/%s/artifacts?artifactName=drop&api-version=5.0-preview.5&%%24format=zip",
```

Comment:
This URL schema is kind of ugly (which isn't our fault). Let's add a link to the Azure Pipeline docs.